cabal-install/tests/UnitTests/Distribution/Client/Init.hs

```diff
@@ -104,7 +134,9 @@ exeFlags = baseFlags {
   -- Create an executable only, with main living in app/Main.hs.
     packageType = Flag Executable
   , mainIs = Flag "Main.hs"
+  , sourceDirs = Just []
```

Comment:
`sourceDirs = Nothing` in `baseFlags` was actually the wrong state for this to be in. Throughout the course of `init` it actually gets turned into `Just []` for `Executable`s. So, to test against this, we set this as our golden data.